packages/@adobe/spectrum-css-temp/components/tabs/index.css

```diff
@@ -31,6 +45,25 @@ governing permissions and limitations under the License.
   vertical-align: top;
 
   outline: none;
+
+  &.spectrum-Tabs-dropdown {
+    inset-inline-start: 0;
+    width: 100%;
+    height: var(--spectrum-tabs-item-height);
+
+    /* TODO: Ask what to do about the below styles. Need to get them propagated to the Picker's FieldButton/Picker */
+    .spectrum-FieldButton {
+      padding: 0 4px;
```

Comment:
urgh unfortunately this doesn't work because the we would also need to get the tab styles applied to the focus ring like `focusRingClass={classNames(styles, 'focus-ring')}`. Will keep trying